views/layouts/main.handlebars

```diff
@@ -1,12 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
 
-    <head>
-        <meta charset="utf-8">
-    </head>
-        <body>
-            <main>
-                {{{body}}}
-            </main>
-        </body>
+<head>
+    <meta charset="utf-8">
+    <title>{{ title }}</title>
+    <link rel="stylesheet" href="/public/css/main.css">
+</head>
+
+<body>
+    <main>
+        <header>
+            <h1 class="title">{{ title }}</h1>
+        </header>
+
+        {{{body}}}
+
+        <footer>
+        </footer>
+    </main>
+    <script src="https://code.jquery.com/jquery-2.2.4.min.js"
```

Comment:
I don't know what the difference between jquery v2 and v3 is but i've been using v3.5.1

Reply:
I might just have an old version. we can update